### ISSUE TITLE
Food in Toffu and a new gamemode called Target

### ISSUE THREAD

**Issue by Comxy** [yogstation13/yogstation#1746]
So you cant make good food if you arnt cook. So I thought. If you can put food in toffu. You have good food without the acces of a 

Also i thought about a new syndicate implant. I dont have a name yet but it alows you to still move if you are in crit. Its about ( 5-10 ) TC

The new gamemode.
You will be targeted by all syndicates and you will start with 20 TC. You will know you are targeted by the other people. But they wont know because they have more objectives. And that is all there is to it. A gamemode that you can play with a less amount of people. Thanks for reading

**Comment by doerakpoes** [comment]
The game mode sounds impossible unless you're Bob Blitzer or all of your hunters are completly uncoördinated and fuck each other up

**Comment by Comxy** [comment]
But they have other objectives too. And they wont know. You know what is comming and you will have the 20 TC to defend yourself

**Comment by doerakpoes** [comment]
So lets say you have 9 people target you and they have completed their other objectives and plan to attack you together and all you have are non slip shoes and a dubble E sword they can all mass disarm you all of and any other gadgets aren't going to help if they attack together

**Comment by Comxy** [comment]
Thats the point of this gamemode. Trying to survive. You can ask securati for help. And you can try to hide. But you need to escape in the shuttle alive. And i dont think there will be 9 traitors. More like 5

**Comment by doerakpoes** [comment]
If sec discovers you have tratoir gear the're just as likley to kill you as your hunters

**Comment by Comxy** [comment]
Come on you get the point. There are more ways of handeling it. Please stop shitting in my post

**Comment by doerakpoes** [comment]
And other then that the targeted could probably just hide inside a deactivated trash can forever and never be found

**Comment by doerakpoes** [comment]
Or change his identity

**Comment by Comxy** [comment]
But he needs to escape on the shuttle or pod alive

**Comment by Comxy** [comment]
So that would just be dumb

**Comment by doerakpoes** [comment]
They'll just hide in the trash can until the time is there

**Comment by doerakpoes** [comment]
Then go sit in a pod and destroy any ine who enters

**Comment by doerakpoes** [comment]
And changing identity helps alot

**Comment by Comxy** [comment]
well. Then we will make the target have other objectives too besides not getting killed

**Comment by doerakpoes** [comment]
I got 1
Objective 2#: don't rage quit get the new Disk X anhilate the station.

**Comment by doerakpoes** [comment]
This game mode just it doesn't feel good it's bassicly wizard but then with traitor items instead of spells

**Comment by matskuman5** [comment]
I dunno, I think theres some promise here

It could be a toggleable setup so you can be a "target" if you want

It could increase tator cooperation cause if they know they have the same assaasinate objective, they know theres a "target" who is heavily armed on the station

**Comment by doerakpoes** [comment]
I still don't like it

**Comment by drderp3635** [comment]
First idea isn't even in fucking English so clarifying that would be nice.

Second idea is just shitty double agents

**Comment by doerakpoes** [comment]
I think he wants to have the ability to put stuff in a toffu just like how you can add stuff to burgers

**Comment by ShadowDeath6** [comment]
We already have stimpacks which let you stay up while in crit. 
Second idea is like double agents but it's less fun for the guy being targeted since he's going to get ganked and unless you stop them from changing IDs which is impossible they automatically make everyone lose because they can just put an agent card on and nobody can get them

**Comment by Comxy** [comment]
Stimpack works different from the implant. The stimpack heals you and gets you out of crit. But the implant alows you to move and still grab stuff. So you can heal yourself. So if you are in space gasping for air and you want to get in. But you fall in crit. With the implant you can still move. Its not like stimpack. Because if you didnt use stimpack it is usless. It just works in a different way and it could save your life.

**Comment by Time-Green** [comment]
What's gonna stop me from going braindead in a random locker and never be found?

**Comment by ShadowDeath6** [comment]
No, stimpacks will keep you from going down while in crit. It was added around a month or so ago.

**Comment by Comxy** [comment]
How about the tofu idea?
